src/doubleTrouble.js

```diff
@@ -26,7 +26,14 @@
  * @return {number[]} - array of remained numbers
  */
 function doubleTrouble(array, target) {
-  // write code here
+  const arrayLength = array.length;
```

Comment:
since you are using arrayLength only in one place I'd say it's not worth extracting it into a variable.

Reply:
done